### ISSUE TITLE
Assist organize file paths

### ISSUE THREAD

**Issue by squld** [kurusugawa-computer/markdown-copilot-vscode#12]
### Proposal
We need a feature to reorganize Markdown files within a workspace to appropriate paths.
This feature can be implemented in three steps:

1. **List** the unmodified diff of the Markdown file paths in the workspace into an editor buffer:
   ```diff
   - path/to/markdown.md
   + path/to/markdown.md
   - other/markdown2.md
   + other/markdown2.md
   ```
2. **Edit** the diff either manually or with Copilot:
   ```diff
   - path/to/markdown.md
   + corrected/path/name.md
   - other/markdown2.md
   + nicer/path/name.md
   ```
3. **Apply** the diff to the file paths.

### Other information
The following considerations are necessary:
- How to infer file names from file contents using Copilot?
- How to organize file paths using Copilot?
- How to determine an order to apply the editing diff?

**Comment by squld** [comment]
I rethought the editing steps for the paths.
It has three steps: **List** -> **Edit** -> **Apply**.
Since it is very difficult to automate the **Edit** step, we will only support manual editing for now.

In the future, when automating the **Edit** step, we can create a command that recommends a path based on the contents of any selected path line.

What do you think?

**Comment by maekawatoshiki** [comment]
> Since it is very difficult to automate the Edit step, we will only support manual editing for now.
> In the future, when automating the Edit step, we can create a command that recommends a path based on the contents of any selected path line.

I agree with the plan putting off *Edit* part. That's hard to create new consistent paths given contents.

**Comment by squld** [comment]
This issue has been resolved in #22
